src/routing/index.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
-
 import { Route } from "react-router-dom";
 import { PrivateRoute } from "./Routes";
 // import { userRoles } from '../services/auth'
 import { PATHS } from "../constant";
-
 import Login from "../pages/Login";
 import CreateClass from "../pages/CreateClass";
 import Course from "../pages/Course";
 import CourseContent from "../pages/CourseContent";
+import ClassList from "../components/Class/ClassList";
```

Comment:
We can also remove this. `import ClassList from "../components/Class/ClassList";`

Reply:
done